hypervisor/util/serial_connection.go

```diff
@@ -28,7 +28,11 @@ func (sc *SerialConnection) AttachSerialConsole(param *hypervisor.ConsoleParam,
 			default:
 				n, err := param.Stdin.Read(b)
 				if err != nil {
-					errc <- errors.Wrap(err, "\nFailed to read from the from the console input buffer\n\n")
+					if err == io.EOF {
+						errc <- errors.Wrap(nil, "\nConsole exited by EOF\n\n")
```

Comment:
The error message at second argument will never be sent back as error.

https://github.com/pkg/errors/blob/master/errors.go#L181-L183

Print as a log message if you need to catch.